dojoWordle/components/Game/Game.style.tsx

```diff
@@ -1,8 +1,41 @@
 import styled from "@emotion/native";
+import { LetterBox, letterStatus } from "../LetterBox.tsx/LetterBox";
+
+export const wordToFind = ["b", "o", "n", "j", "o", "u", "r"];
 
 export const ResultContainer = styled.View({
   alignItems: "center",
   flexDirection: "row",
   justifyContent: "center",
   paddingBottom: 50,
 });
+
+export const displayCheckedWord = (wordGuessed: string) => {
+  var wordWithStatus = [];
+  var i: number;
+
+  {
+    for (i = 0; i < wordToFind.length; i++) {
+      var status: letterStatus;
+      status = "failure";
+
+      if (wordToFind[i] === wordGuessed[i]) {
+        status = "success";
+      } else if (wordToFind.find((character) => character === wordGuessed[i])) {
+        status = "missed";
+      }
+      wordWithStatus[i] = <LetterBox status={status} letter={wordGuessed[i]} />;
```

Comment:
You should add a key to LetterBox component if you display a list.

See: https://beta.reactjs.org/learn/rendering-lists#rules-of-keys
and: https://beta.reactjs.org/learn/rendering-lists#why-does-react-need-keys